### PR TITLE
chore: Adding local testing steps to contributing guide

### PR DESCRIPTION
**Pull request by ddixit14**
Hi team, just adding some stuff to the guide that maybe useful for contributors.

@suztomo Opened this new PR because I was unable to edit the last one, looks like I messed up with the head of that branch while working on the versioning strategy. Also, there are no steps to generate release-please command afaik. It is included in the node build command. Multi-line command is fixed.

image

@chingor13 can you please approve this one?

### CODE REVIEW

**Comment by suztomo** [comment]
> node build command

Write down that.

**Comment by ddixit14** [comment]
> > node build command
> 
> Write down that.

I have already done that. 

A074C7D9-0B36-4092-ABBE-4C12B339C506

**Comment by suztomo** [comment]
@ddixit14 That 2nd part, "node build/src .." command, seems correct. But the 1st part, "release-please release-pr .." command, does not use the some new code without the command is rebuilt.

**Comment by ddixit14** [comment]
> @ddixit14 That 2nd part, "node build/src .." command, seems correct. But the 1st part, "release-please release-pr .." command, does not use the some new code without the command is rebuilt.

I don't understand what you're trying to say. 
The first command ""release-please release-pr .." is your normal CLI command. This is just an example of a release-please CLI command (this is not related to building from source or anything). 
You can run the equivalent of this CLI command from your source (after doing some code changes) using the 2nd command.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,21 +59,21 @@ accept your pull requests.
 
 1. After you've written some new code, in order to test it out, you can use the [CLI][CLI].
 
-   For example, the CLI command:
+   The below command should be run from the root of the source code:
 
    ```
-   release-please release-pr \
+   node build/src/bin/release-please.js release-pr \
```
Comment (suztomo) [inline CONTRIBUTING.md:65]: Now it's clear. Thank you.

**Review by suztomo — APPROVED** [review]

**Review by bcoe — APPROVED** [review]

**Review by bcoe — APPROVED** [review]

**Comment by suztomo** [comment]
@bcoe Can you merge this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,4 +55,25 @@ accept your pull requests.
...
-[node]: https://nodejs.org/en/
\ No newline at end of file
+## Testing a new feature using CLI
+
+1. After you've written some new code, in order to test it out, you can use the [CLI][CLI].
+
+   The below command should be run from the root of the source code:
+
+   ```
+   node build/src/bin/release-please.js release-pr \
```
Comment (suztomo) [inline CONTRIBUTING.md:65]: build/src/bin/release-please.js? Not typescript?

**Pull request merged**